<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1453</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Basic Camera Feed</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QWidget" name="camera_widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>701</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>640</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Alignment</string>
    </property>
   </widget>
   <widget class="QWidget" name="processing_widget" native="true">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>701</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
   <widget class="QPushButton" name="up_button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>630</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="down_button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>670</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="left_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>650</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="right_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>650</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="live_button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>670</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>live</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>630</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="minus_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>670</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="laser_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>710</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>laser</string>
    </property>
   </widget>
   <widget class="QPushButton" name="laser_process_button">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>690</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>process_laser</string>
    </property>
   </widget>
   <widget class="QPushButton" name="live_button_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>700</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>live</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_button">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>710</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="linesave_button">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>660</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ROI_fiber_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>640</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ROI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="topled_button">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>660</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>top led</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="save_edit">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>660</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>line_image_</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ROI_mic_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>670</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ROI</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1453</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHello">
    <property name="title">
     <string>Hello</string>
    </property>
   </widget>
   <addaction name="menuHello"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
